datafusion/core/src/datasource/physical_plan/csv.rs

```diff
@@ -529,6 +556,7 @@ impl CsvOpener {
         config: Arc<CsvConfig>,
         file_compression_type: FileCompressionType,
     ) -> Self {
+        dbg!(&config);
```

Comment:
This one, probably, should be cleaned up too

Reply:
Thanks, I missed to remove them 😢